docs/website/docs/dlt-ecosystem/verified-sources/rest_api.md

```diff
@@ -576,3 +576,18 @@ See the [incremental loading](../../general-usage/incremental-loading.md#increme
 - `root_key` (bool): Enables merging on all resources by propagating root foreign key to child tables. This option is most useful if you plan to change write disposition of a resource to disable/enable merge. Defaults to False.
 - `schema_contract`: Schema contract settings that will be applied to this resource.
 - `spec`: A specification of configuration and secret values required by the source.
+
+## Troubleshooting
+
+If you encounter any issues while setting up or running the pipeline, check the following:
+
+- Make sure you have the correct access token and other credentials in the `secrets.toml` file.
+- Check the API documentation for the correct endpoint paths and query parameters.
```

Comment:
When does that happen? What is the symptom? I assume 404? Also ask the user to check if base url is ok and if ask them to verify what they get when they combine it with path. is anything missing?
also if the path starts with `/path` - the path in base url is ignored. it is a common mistake worth mentioning

Reply:
> also if the path starts with /path - the path in base url is ignored. it is a common mistake worth mentioning

@rudolfix could you elaborate? I don't think it the base url would be ignored. That's why we have a custom [join_url](https://github.com/dlt-hub/dlt/blob/devel/dlt/sources/helpers/rest_client/utils.py#L1)

Reply:
this should be done as well